extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -1,32 +1,38 @@
-import {Layout_Enum, applyFillContent} from '#core/dom/layout';
+import * as Preact from '#core/dom/jsx';
+import {Layout_Enum} from '#core/dom/layout';
 
-import {AmpStoryPageAttachment} from 'extensions/amp-story/1.0/amp-story-page-attachment';
-
-const TAG = 'amp-story-shopping-attachment';
-
-export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
+export class AmpStoryShoppingAttachment extends AMP.BaseElement {
```

Comment:
Since `amp-story-shopping-attachment` will be the parent element of its `amp-story-page-attachment` child instead of `amp-story`, I'm curious whether [startListeningForTouchEvents_()](https://github.com/ampproject/amphtml/blob/22d6da19b9e01e75e6b67f54cd425c465822fe8c/extensions/amp-story/1.0/amp-story-draggable-drawer.js#L241-L248) will work as expected because its comment suggests that swiping up is only allowed when the attachment is a direct descendant of `amp-story`. Mind verifying locally that opening and closing an `amp-story-shopping-attachment` works as expected?

Reply:
Good catch, that is broken. Looking into it now.

Reply:
I added an exception in `amp-story-draggable-drawer.js`. lmkwut